### PR TITLE
Upgrade containerd to fix high security severity issue(CVE-2022-23648)

### PR DESCRIPTION
**Pull request by dpericaxon**
Signed-off-by: Dan Peric <dperic@axon.com>

**Description:** 
Updated containerd dependency to 1.16.1 to mitigate CVE-2022-23648
https://nvd.nist.gov/vuln/detail/CVE-2022-23648

### CODE REVIEW

File: cmd/configschema/go.mod [hunk 1]
```diff
@@ -62,7 +62,6 @@ require (
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v1.8.5 // indirect
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/internal/resourcemapping v0.32.5 // indirect
 	github.com/Microsoft/go-winio v0.5.2 // indirect
-	github.com/Microsoft/hcsshim v0.9.2 // indirect
```
Comment (jpkrohling) [inline cmd/configschema/go.mod:65]: Did you just manually remove this? Or did you run go mod tidy? I believe @codeboten ran a go mod tidy recently, which should have addressed this, if removing is the right thing to do...

Comment (jpkrohling) [reply cmd/configschema/go.mod:65]: @codeboten, shouldn't your recent "go mod tidy" PRs have addressed this?

**Comment by jpkrohling** [comment]
While I'm fine with having newer versions of containerd , it doesn't look like we depend on it. I don't think changing the `// indirect` entries in go.mod is appropriate either. But perhaps I'm missing something?

**Comment by locmai** [comment]
The `indirect` tags got changed by the root Makefile step `gotidy` - the _recent "go mod tidy" PRs_  they were only applied for a few receivers while this one would do for all. So in the release build, we still have the vulnerable packages pulled in. Unless you want to split this up per receiver?

**Comment by dmitryax** [comment]
@dpericaxon you can apply go mod replace rule instead

**Comment by jpkrohling** [comment]
I'm closing this, if you think it's not appropriate, let me know.

**Pull request closed without merging**